Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -69,9 +103,58 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
+  private func configureViews() {
+    // Configure subviews
+    self.backThisProjectContainerView.addSubview(self.backThisProjectButton)
+
+    self.view.addSubview(self.backThisProjectContainerView)
+    self.view.bringSubviewToFront(backThisProjectContainerView)
+
+    self.backThisProjectButton.addTarget(self, action: #selector(backThisProjectTapped), for: .touchUpInside)
+
+    // Configure constraints
+    // swiftlint:disable line_length
+    let backThisProjectContainerViewConstraints = [self.backThisProjectContainerView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
+                                                   self.backThisProjectContainerView.rightAnchor.constraint(equalTo: self.view.rightAnchor),
+                                                   self.backThisProjectContainerView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
+    ]
+    let containerMargins = self.backThisProjectContainerView.layoutMarginsGuide
+
+    let backThisProjectButtonConstraints = [self.backThisProjectButton.leftAnchor.constraint(equalTo: containerMargins.leftAnchor),
+                                            self.backThisProjectButton.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
+                                            self.backThisProjectButton.bottomAnchor.constraint(equalTo: containerMargins.bottomAnchor),
+                                            self.backThisProjectButton.topAnchor.constraint(equalTo: containerMargins.topAnchor),
+                                            self.backThisProjectButton.heightAnchor.constraint(equalToConstant: Styles.minTouchSize.height)
+    ]
+
+    NSLayoutConstraint.activate([backThisProjectContainerViewConstraints,
+                                 backThisProjectButtonConstraints].flatMap { $0 })
+  }
+
+  public override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.backThisProjectContainerView
+      |> \.layoutMargins .~ .init(all: backThisProjectContainerViewMargins)
+
+    _ = self.backThisProjectButton
+      |> greenCTAButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in
+        return Strings.project_back_button()
+    }
+  }
+
   public override func bindViewModel() {
     super.bindViewModel()
 
+    self.viewModel.outputs.goToRewards
+      .observeForControllerAction()
+      .observeValues { [weak self] params in
+        let (project, refTag) = params
```

Comment:
Is there a reason we won't be needing `reward` in this case?

Reply:
All the rewards are in `project.rewards`!